.travis.yml

```diff
@@ -11,18 +12,39 @@ install: true
 cache:
   directories:
     - node_modules
+    
 jobs:
   include:
-    - stage: "Prepare Dependencies"
+    - stage: Mac Setup and Tests
+      os: osx
+      name: "Prepare Dependencies"
+      script:
+        - npm install
+        - npm update
+    - <<: *Mac-Setup
+      os: osx
+      name: "Lint"
+      script:
+        - npm run lint
+    - <<: *Mac-Setup
+      os: osx
+      name: "Jasmine"
+      script:
+        - npm run test
+
+    - stage: Linux Setup
```

Comment:
1 very small nitpick.

The stage for Mac is called "Mac Setup and Tests". Maybe this one can be "Linux Setup and Tests" too.

Reply:
Alrights!